config.json

```diff
@@ -264,75 +264,147 @@
       {
         "groupId": "androidx.compose.animation",
         "artifactId": "animation",
-        "version": "1.4.3",
-        "nugetVersion": "1.4.3.2",
+        "version": "1.5.0",
+        "nugetVersion": "1.5.0",
         "nugetId": "Xamarin.AndroidX.Compose.Animation",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.compose.animation",
+        "artifactId": "animation-android",
+        "version": "1.5.0",
+        "nugetVersion": "1.5.0",
+        "nugetId": "Xamarin.AndroidX.Compose.Animation.Android",
+        "dependencyOnly": false
+      },
       {
         "groupId": "androidx.compose.animation",
         "artifactId": "animation-core",
-        "version": "1.4.3",
-        "nugetVersion": "1.4.3.2",
+        "version": "1.5.0",
+        "nugetVersion": "1.5.0",
         "nugetId": "Xamarin.AndroidX.Compose.Animation.Core",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.compose.animation",
+        "artifactId": "animation-core-android",
+        "version": "1.5.0",
+        "nugetVersion": "1.5.0",
+        "nugetId": "Xamarin.AndroidX.Compose.Animation.Core.Android",
+        "dependencyOnly": false
+      },
       {
         "groupId": "androidx.compose.animation",
         "artifactId": "animation-graphics",
-        "version": "1.4.3",
-        "nugetVersion": "1.4.3.2",
+        "version": "1.5.0",
+        "nugetVersion": "1.5.0",
         "nugetId": "Xamarin.AndroidX.Compose.Animation.Graphics",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.compose.animation",
+        "artifactId": "animation-graphics-android",
+        "version": "1.5.0",
+        "nugetVersion": "1.5.0",
+        "nugetId": "Xamarin.AndroidX.Compose.Animation.Graphics.Android",
```

Comment:
I think this one is missing:
```
<remove-node path="/api/package" />
```

Reply:
There were several Metadata files missing, so these were not included in projects and thus no transformation happened and MCWs were created. 

I have added metadata and now it should be OK.